source/extensions/filters/http/lua/lua_filter.cc

```diff
@@ -11,6 +11,7 @@
 #include "source/common/config/datasource.h"
 #include "source/common/crypto/utility.h"
 #include "source/common/http/message_impl.h"
+#include "source/common/router/config_impl.h"
```

Comment:
No need to include the impl here or in the BUILD file above, same elsewhere if I missed it.